docs/about/whats-new/whats-new-0.6.0.ipynb

```diff
@@ -0,0 +1,6969 @@
+       "       &#x27;2021-01-01T12:05:07&#x27;, &#x27;2021-01-01T12:05:08&#x27;,\n",
+       "       &#x27;2021-01-01T12:05:09&#x27;, &#x27;2021-01-01T12:05:10&#x27;,\n",
+       "       &#x27;2021-01-01T12:05:11&#x27;, &#x27;2021-01-01T12:05:12&#x27;,\n",
+       "       &#x27;2021-01-01T12:05:13&#x27;, &#x27;2021-01-01T12:05:14&#x27;,\n",
+       "       &#x27;2021-01-01T12:05:15&#x27;, &#x27;2021-01-01T12:05:16&#x27;,\n",
+       "       &#x27;2021-01-01T12:05:17&#x27;, &#x27;2021-01-01T12:05:18&#x27;,\n",
+       "       &#x27;2021-01-01T12:05:19&#x27;, &#x27;2021-01-01T12:05:20&#x27;,\n",
+       "       &#x27;2021-01-01T12:05:21&#x27;, &#x27;2021-01-01T12:05:22&#x27;,\n",
+       "       &#x27;2021-01-01T12:05:23&#x27;, &#x27;2021-01-01T12:05:24&#x27;,\n",
+       "       &#x27;2021-01-01T12:05:25&#x27;, &#x27;2021-01-01T12:05:26&#x27;,\n",
+       "       &#x27;2021-01-01T12:05:27&#x27;, &#x27;2021-01-01T12:05:28&#x27;,\n",
+       "       &#x27;2021-01-01T12:05:29&#x27;, &#x27;2021-01-01T12:05:30&#x27;,\n",
+       "       &#x27;2021-01-01T12:05:31&#x27;, &#x27;2021-01-01T12:05:32&#x27;,\n",
+       "       &#x27;2021-01-01T12:05:33&#x27;, &#x27;2021-01-01T12:05:34&#x27;,\n",
+       "       &#x27;2021-01-01T12:05:35&#x27;, &#x27;2021-01-01T12:05:36&#x27;,\n",
+       "       &#x27;2021-01-01T12:05:37&#x27;, &#x27;2021-01-01T12:05:38&#x27;,\n",
+       "       &#x27;2021-01-01T12:05:39&#x27;, &#x27;2021-01-01T12:05:40&#x27;,\n",
+       "       &#x27;2021-01-01T12:05:41&#x27;, &#x27;2021-01-01T12:05:42&#x27;,\n",
+       "       &#x27;2021-01-01T12:05:43&#x27;, &#x27;2021-01-01T12:05:44&#x27;,\n",
+       "       &#x27;2021-01-01T12:05:45&#x27;, &#x27;2021-01-01T12:05:46&#x27;,\n",
+       "       &#x27;2021-01-01T12:05:47&#x27;, &#x27;2021-01-01T12:05:48&#x27;,\n",
+       "       &#x27;2021-01-01T12:05:49&#x27;, &#x27;2021-01-01T12:05:50&#x27;,\n",
+       "       &#x27;2021-01-01T12:05:51&#x27;, &#x27;2021-01-01T12:05:52&#x27;,\n",
+       "       &#x27;2021-01-01T12:05:53&#x27;, &#x27;2021-01-01T12:05:54&#x27;,\n",
+       "       &#x27;2021-01-01T12:05:55&#x27;, &#x27;2021-01-01T12:05:56&#x27;,\n",
+       "       &#x27;2021-01-01T12:05:57&#x27;, &#x27;2021-01-01T12:05:58&#x27;,\n",
+       "       &#x27;2021-01-01T12:05:59&#x27;, &#x27;2021-01-01T12:06:00&#x27;,\n",
+       "       &#x27;2021-01-01T12:06:01&#x27;, &#x27;2021-01-01T12:06:02&#x27;],\n",
+       "      dtype=&#x27;datetime64[s]&#x27;)</pre></li></ul></li></ul></div></div>"
+      ],
+      "text/plain": [
+       "  <scipp.Variable>          datetime64  [s]              (time)  [2021-01-01T12:02:03, 2021-01-01T12:02:04, ..., 2021-01-01T12:06:01, 2021-01-01T12:06:02]"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "var = sc.array(dims=['time'], values=np.arange(np.datetime64('2021-01-01T12:00:00'), np.datetime64('2021-01-01T12:04:00')))\n",
```

Comment:
i would add a new line between the two `np.datetime` as the line is a little long when rendered in the notebook.